lib/ts/recipe/emailpassword/emaildelivery/services/smtp/index.ts

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+    ServiceInterface,
+    GetContentResult,
+    SMTPInputConfig,
+    getSMTPProvider,
+} from "../../../../emaildelivery/services/smtp";
+import { Transporter } from "nodemailer";
+import { TypeEmailPasswordEmailDeliveryInput } from "../../../types";
+import getPasswordResetEmailContent from "./passwordReset";
+import getEmailVerifyEmailContent from "../../../../emailverification/emaildelivery/services/smtp/emailVerify";
+
+export default function getEmailPasswordEmailServiceSMTP(config: SMTPInputConfig<TypeEmailPasswordEmailDeliveryInput>) {
```

Comment:
- This function should be exposed in `emailpassword/emailDeilvery/smtp/index.ts`
- Also, rename it to `getSMTPService`